File: 0485-max-consecutive-ones/0485-max-consecutive-ones.js
const findMaxConsecutiveOnes = (nums) => {
  let w = 0, m = 0;
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] === 1) {
      w++;
      if (m < w) m = w;
    } else {
      w = 0;
    }
  }
  return m;
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.go
func findMaxConsecutiveOnes(nums []int) int {
  ans := 0
  cnt := 0
  for _, num := range nums {
    if num == 1 {
      cnt += 1
    } else {
      if cnt > ans {
        ans = cnt
      }
      cnt = 0
    }
  }
  if cnt > ans {
    ans = cnt
  }
  return ans
}